### PR TITLE
DOCSP-22748 In-Use Encryption Tutorial 

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Walk through of using In-Use Encryption in Compass.

JIRA: https://jira.mongodb.org/browse/DOCSP-22748

STAGING: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-22748/in-use-encryption-tutorial/

### CODE REVIEW

File: source/in-use-encryption-tutorial.txt [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+   :class: singlecol
+
+In-Use Encryption allows you to connect your deployments using :v6.0:`Queryable 
+Encryption </core/queryable-encryption/>`. This connection method allows you to 
+encrypt a subset of fields in your collections. 
+
+Overview
+--------
+
+This guide shows you how to connect your deployment and collections using
```
Comment (jmd-mongo) [inline source/in-use-encryption-tutorial.txt:22]: [question]

Should this be "connect to your deployment"?

File: source/in-use-encryption-tutorial.txt [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+When In-Use Encryption is **enabled**:
+
+- You can modify encrypted values.
+
+- You can insert documents and specified fields will be encrypted.
+
+When In-Use Encryption is **disabled**:
+
+- You can no longer modify encrypted values. |compass-short| displays the values
```
Comment (jmd-mongo) [inline source/in-use-encryption-tutorial.txt:128]: [suggestion]

Consider:

"You cannot modify encrypted values."

File: source/in-use-encryption-tutorial.txt [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+Requirements and Limitations
+----------------------------
+
+- In-Use Encryption is an Enterprise/Atlas only feature. 
+
+- You need a replica set to use this connection option. Your replica set can be 
+  a single node or larger.
+
+- You need to connect your deployment on Compass using In-Use Encryption. For 
+  more information on how to connect your deployment, see :ref:`In-Use Encryption
```
Comment (jmd-mongo) [inline source/in-use-encryption-tutorial.txt:38]: Similar question as on line 15.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the changes, @jocelyn-mendez1! I've left some suggestion and questions for your consideration.

Thanks!
Joe

File: source/in-use-encryption-tutorial.txt (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/in-use-encryption-tutorial.txt:38]: Done.

File: source/in-use-encryption-tutorial.txt [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+- You can modify encrypted values.
+
+- You can insert documents and specified fields will be encrypted.
+
+When In-Use Encryption is **disabled**:
+
+- You can no longer modify encrypted values. |compass-short| displays the values 
+  of these fields as a series of asterisks.
+
+- Inserted documents will no longer encrypt fields.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/in-use-encryption-tutorial.txt]: Done.

File: source/in-use-encryption-tutorial.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/in-use-encryption-tutorial.txt]: Done.

File: source/in-use-encryption-tutorial.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/in-use-encryption-tutorial.txt]: Done.

File: source/in-use-encryption-tutorial.txt [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+Requirements and Limitations
+----------------------------
+
+- In-Use Encryption is an Enterprise/Atlas only feature. 
+
+- You need a replica set to use this connection option. Your replica set can be 
+  a single node or larger.
+
+- You need to connect to your deployment on Compass using In-Use Encryption. For 
+  more information on how to connect your deployment, see :ref:`In-Use Encryption
```
Comment (jmd-mongo) [inline source/in-use-encryption-tutorial.txt:38]: [nit]

One minor bit here -- I'd suggest "how to connect to your deployment" on line 38.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @jocelyn-mendez1! This one LGTM mod a tiny nit.

Thank you!
Joe

File: source/in-use-encryption-tutorial.txt (hunk 5 above)
Comment (jocelyn-mendez1) [reply source/in-use-encryption-tutorial.txt]: Done.

**Review by addaleax — APPROVED** [review]

File: source/in-use-encryption-tutorial.txt [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In-Use Encryption allows you to connect to your deployments using :v6.0:`Queryable 
+Encryption </core/queryable-encryption/>`. This connection method allows you to
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/in-use-encryption-tutorial.txt:16]: I agree! I removed the note, thank you!

**Review by addaleax — APPROVED** [review]

**Pull request merged**